.php_cs.dist

```diff
@@ -0,0 +1,11 @@
+<?php 
+
+use Gpupo\CommonDevExtra\CsConfigurator;
```

Comment:
Where is this package required? And what does it do?

Reply:
this is required at composer global env and that set the php-cs-fixer config. If the project adopt that changes, we can replace with pure php-cs-fixer config

Reply:
assertEquals does not take datatype into consideration and type is one big focus of new level of php :)

Reply:
@mghoneimy i propose to make a release with #46 and after, change the composer.json to php8 only, to give to the project things like named parameters and better types. A new major version, so then with this PR